cg/cli/demultiplex/copy_novaseqx_demultiplex_data.py

```diff
@@ -114,3 +117,103 @@ def is_ready_for_post_processing(flow_cell_dir: Path, demultiplexed_runs_dir: Pa
         flow_cell_is_ready = False
 
     return flow_cell_is_ready
+
+
+def get_existing_manifest_file(source_directory: Path) -> Path | None:
```

Comment:
I just moves these functions into this directory. They will likely need a better place at some point

Reply:
I agree